packages/adapters/solana/src/client.ts

```diff
@@ -192,7 +192,7 @@ export class SolanaAdapter extends AdapterBlueprint<SolanaProvider> {
       provider,
       connection,
       to: params.to,
-      value: params.value as number
+      value: Number(params.value)
```

Comment:
Should we check `NaN` case here? What happens if somehow value is not a valid  number?

Reply:
good one !